client/src/components/PacmanGame/PacmanGame.jsx

```diff
@@ -24,35 +28,87 @@ class PacmanGame extends Component {
       for (const [x, y] of entityLocations) {
         gridState[x][y] = entityCode;
       }
-      // return gridState;
     };
 
-
-    // apply entities duummy data
     entityApplier(getFoods(), entityToCode('food'));
     entityApplier(getPacman(), entityToCode('pacman'));
-    console.log((gridState[43][5]));
-    // console.log(gridState[43])
     entityApplier(getWalls(cellsInEachRow), entityToCode('wall'));
     entityApplier(getGhosts(), entityToCode('ghost'));
-    console.log((gridState[43][5]));
     entityApplier(getEnergizers(), entityToCode('energizer'));
 
-    this.setState({ 'gridState': gridState });
-    this.setState({ 'gridSize': gridSize });
+    this.setState({ gridState, gridSize });
 
-    console.log(getPacman());
-    console.log((gridState[43][5]));
-    console.log('food', (gridState[15][9]));
+    let walls = [];
+    gridState.forEach((rowArray, rowIndex) => {
+      rowArray.forEach((cell, columnIndex) => {
+        if (cell === 4) {
```

Comment:
Don't use 4 directly as it is difficult to remember what it entity it represents. Use the `entityToCode('wall')` instead to make this more readable.